src/v2/server.ts

```diff
@@ -56,15 +55,15 @@ app.get(
       }
 
       const headTagsString = ReactDOM.renderToString(headTags as any)
-      const sharifyData = res.locals.sharify.script()
+      const sharify = res.locals.sharify
 
       const { APP_URL, CURRENT_PATH, WEBFONT_URL } = sharify.data
```

Comment:
The update turns out to fix the `CURRENT_PATH` that `apple-itunes-app` meta tag relies on. See below for before and after:

![Screen Shot 2021-09-07 at 1 47 10 PM](https://user-images.githubusercontent.com/796573/132390621-e5475912-7436-49c2-8d99-03cc3f3840d4.png)